schemas.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Optional
+from pydantic import BaseModel
+from datetime import date
```

Comment:
Add blank lines between imports that belong to different import groups in all files to follow PEP-8